src/Model/SnapshotChildrenCollection.php

```diff
@@ -13,48 +13,40 @@
 
 namespace Sonata\PageBundle\Model;
 
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+
 /**
  * SnapshotChildrenCollection.
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
- *
- * @final since sonata-project/page-bundle 3.26
  */
-class SnapshotChildrenCollection implements \Countable, \IteratorAggregate, \ArrayAccess
+final class SnapshotChildrenCollection implements \Countable, \IteratorAggregate, \ArrayAccess
 {
-    /**
-     * @var TransformerInterface
-     */
-    protected $transformer;
+    private TransformerInterface $transformer;
 
-    /**
-     * @var PageInterface
-     */
-    protected $page;
+    private PageInterface $page;
 
-    /**
-     * @var array
-     */
-    protected $collection;
+    private ?Collection $collection = null;
```

Comment:
```suggestion
    /**
     * @var array
     */
    private $collection;
```
Let's this untouched, since it's broken so far.

Reply:
done